apps/status-api/README.md

```diff
@@ -8,12 +8,29 @@ DeFiChain Status API, providing the statuses of different DeFiChain services
 
 To decouple the DeFiChain products from the status page, the approach of having a centralised provider to determine each DeFiChain service status with a pre-defined logic will allow it to be maintained consistently throughout. This will allow other apps or services to share the status from the same Status APIs.
 
-### `/blockchain/status`
+### `/blockchain`
 >https://github.com/DeFiCh/jellyfish/issues/1271
 
 To provide the status of the blockchain based on the block creation time interval
+
+| Status             | Threshold Time    |
+|--------------------|-------------------|
+| `operational`      | `< 30 minutes`    |
+| `degraded`         | `30 - 45 minutes` |
+| `outage`           | `> 45 minutes`    |
+
+### `/oracles?address=<address>`
+
+To provide the status of each oracle given the address based on the last published time for any given token
```

Comment:
Based on the written statement below, "address" isn't an optional parameter; shouldn't address in `/oracles/:address` be part of the path instead?

Unless the scenario here is: given address is optional, we check the health of all oracles?